piptools/sync.py

```diff
@@ -14,8 +14,14 @@
     'pip-tools',
     'pip-review',
     'pkg-resources',
+
+    # stdlib and dev packages
     'setuptools',
     'wheel',
+    'distribute',
+    'python',
+    'wsgiref',
+    'argparse',
```

Comment:
argparse is not stdlib package on Python <=2.6 (or 3.0 or 3.1). See https://pypi.org/project/argparse/

Reply:
According to `setup.py` pip-tools doesn't support python 2.6/3.0/3.1 `python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"`. Should i fix it?